frontend/src/Components/Account/Register.js

```diff
@@ -1,5 +1,159 @@
+              />
+            </div>
+            <div className="mb-4">
+              <label
+                className="block text-gray-700 text-sm font-bold mb-2"
+                htmlFor="email"
+              >
+                Name
+              </label>
+              <input
+                aria-label="name"
+                name="name"
+                value={form.name}
+                type="name"
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="Name..."
+              />
+            </div>
+            <div className="mb-4">
+              <label
+                className="block text-gray-700 text-sm font-bold mb-2"
+                htmlFor="email"
+              >
+                Email
+              </label>
+              <input
+                aria-label="user name"
+                name="email"
+                value={form.email}
+                type="email"
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="Email"
+              />
+            </div>
+            <div className="mb-2">
+              <label
+                className="block text-gray-700 text-sm font-bold mb-2"
+                htmlFor="password"
+              >
+                Password
+              </label>
+              <input
+                aria-label="Password"
+                name="password"
+                type="password"
+                value={form.password}
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="******************"
+              />
+            </div>
+
+            <div className="mb-4">
+              <label
+                className="block text-gray-700 text-sm font-bold mb-2"
+                htmlFor="email"
+              >
+                Description
+              </label>
+              <textarea
+                aria-label="description"
+                name="description"
+                value={form.description}
+                type="description"
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="Describe Yourself..."
+              />
+            </div>
+
+            {formError && (
+              <div className="text-red-400 font-semibold">
+                Invalid Credentials
```

Comment:
Sign Up doesn't need to have Invalid Credentials, we can think about it once we have a Confirm Password? Probably best will be handled by validators https://github.com/Open-Sourced-Olaf/SocioMark/issues/24